app/build.gradle

```diff
@@ -413,6 +413,7 @@ dependencies {
     implementation Deps.mozilla_support_ktx
     implementation Deps.mozilla_support_rustlog
     implementation Deps.mozilla_support_utils
+    implementation Deps.mozilla_support_migration
```

Comment:
nit: Lets only add this for the `fennecProduction` build type. This will also make sure that we are not landing anything in "regular fenix" by mistake. :)

Reply:
Good call!